### DISCUSSION TITLE
How 'Area' type chart has points at reverse point?

### DISCUSSION THREAD

**Discussion by twilighthook** [TanStack/react-charts#85] [General] [answered]
Line chart has 'showPoints' prop can show.
How Area chart show the points without modify the source code.
Or maybe the only way is add showPoints prop and return Point component in Area component?

**Comment by evenstephenr** [comment]
@twilighthook you should be able to use the `showPoints` control on an Area series as well as a line series, example below

```
...

  const series = React.useMemo(
    () => ({
      type: "area",
      showPoints: true,
    }),
    []
  );

...
```

Does this solve the issue for you?

**Comment by tannerlinsley** [comment, marked as answer]
Due to a massive overhaul and rewrite of React Charts to TypeScript and a massively upgraded public API, this discussion is being marked as answered for triage. The latest version can be tested via the `react-charts@beta` tag. Documentation is in the process of being upgraded, but thankfully, everything is fully typed now, so you can explore the API that way for now.

If, *after testing or playing with the v3 beta*, you believe this discussion is still relevant or unanswered, feel free to mark it as such and continue the discussion.
